rfc/rfc-0015-supply-chain-selection-through-traits.md

```diff
@@ -1,3 +1,157 @@
-[~~Draft PR~~](https://github.com/vmware-tanzu/cartographer/pull/)
+# Draft RFC <0000> <Title>
 
-[Draft Text](https://github.com/vmware-tanzu/cartographer/blob/rfc-0015-supply-chain-selection-through-traits/rfc/rfc-0015-supply-chain-selection-through-traits.md)
+## Summary
+
+A mechanism to allow a cluster to have multiple Supply Chains declared
+1. With similar purposes (or where one extends the behaviour of another)
+2. That will match the developer's requirements without the developer needing to understand naming permutations.
+
+
+```gherkin
+Given Supply Chains that represent permutations of a behaviour tree
+When A developer annotates their workload to select for a permutation
+Then the correct supply chain is selected.
+```
+
+## Motivation
+
+Examples of app characteristics that might make up a permutation:
+
+* `type`:
+    * `web`: an app that will deploy as a web service endpoint
+    * `agent`: an app that monitors other services
+
+* `input`:
+    * `image`: an app that is provided as a OCI image
+    * `source`: an app with source
+
+* `test`:
+    * `tekton`: the devloper has a tekton pipeline that can be used to test the code
+
+* `scan`:
```

Comment:
Suggest we remove scanning, given that it's more of a policy than a capability.

Eventually, we could create validating/mutating admission webhooks for Workloads that would allow this feature to be used with policies.